Funcky/Monads/Either/Either.Core.cs

```diff
@@ -104,6 +93,19 @@ public override string ToString()
         => Match(
             left: static left => $"Left({left})",
             right: static right => $"Right({right})");
+
+    private bool TryGetRight([NotNullWhen(true)] out TRight? right, [NotNullWhen(false)] out TLeft? left)
```

Comment:
I think the name TryGetRight implies another method, which only has the right out parameter, this is not the usual Try* Pattern.